flathub.json

```diff
@@ -1,4 +1,6 @@
 {
+  "automerge-flathubbot-prs": false,
```

Comment:
Shout we auto-merge?

Reply:
im really not sure.  i wanna say yes, but maybe we should defer to @fpiesche